magni_bringup/README.md

```diff
@@ -1,40 +1,48 @@
 # Magni bringup
 
-History:
-We decided to move many motor controller and other key operating mode parameters from base.yaml to robot.yaml.
-A lot of work was done to do this by Rohan but the issue has been only creaping along and needs closure so that it will finally be safe to use magni_robot repository along with ubiquity_motor and our main codeline will then be more supportable again.
-
 **Robot startup procedure:**
 Magni robots start ros core and a set of base nodes on boot. This is done in the following order:
-1.) using `systemctl` the magni-base service file (`/usr/sbin/magni-base.service`) is ran. This:
-  a) starts roscore,
-  b) sets up environment variables,
-  c) ..., TODO what else is being launched there?
-  d) launches `base.launch`
-2.) `base.launch` launches robot base nodes:
-  a) `logitech.launch`
-  b) `rosbridge.launch` 
-  c) placeholder for other nodes that are NOT VARIABLE (don't change with robot config)
-  d) starts `launch_core.py`
-3.) `launch_core.py` creates and launches `core.launch` (by default in `/tmp/core.launch`) which starts up the rest of the robot nodes that are variable (change with robot config):
-  a) motor node: runs ubiquity_motor node with the parameters extracted from either `/etc/ubiquity/robot.yaml` OR `magni_bringup/param/robot.yaml` with respective priorities
-  b) robot description: all robot static and dynamic transforms. It gets its camera and lidar extrinsics from either `~/.ros/extrinsics/<SENSOR>_extrinsics_<POSITION>.yaml` OR `magni_description/extrinsics/<SENSOR>_extrinsics_<POSITION>.yaml` with respective priorities. 
-  c) other nodes like `controller_spawner`, `diagnostic_aggregator`, `oled_display_node`,... Exactly what gets launched can be seen in `magni_bringup/param/core_launch.em` from which core.launch is generated. Any addition to core.launch should be added inside the `core_launch.em` file
 
-**Useful commands for debugging**
-If new additions need to be added to core.launch, add them into `magni_bringup/param/core_launch.em`, then edit the launch_core.py accordingly. You can then run the commands:
+`systemctl start magni-base.service`:
+  - roscore
```

Comment:
roscore is a separate systemd service roscore.service

Reply:
fixed